public/employers/buzzfeed.yml

```diff
@@ -0,0 +1,34 @@
+name: BuzzFeed
+wiki: BuzzFeed
+image: buzzfeed.svg
+officialWebsite: https://www.buzzfeed.com/about
+industries:
+  - entertainment
+  - media
+location:
+  city: New York City
+  state: NY
+  country: us
+  international: false
+  wiki: New York City
+employeesBefore:
+  type: approximately
+  upperBound: 1700
+  year: 2017
+  yearQuarter: Q4
+summary: BuzzFeed announced plans to avoid layoffs by significantly cutting executive salaries and giving gradual pay cuts to other salaried employees.
```

Comment:
They may have announced that, but they still laid off people. https://variety.com/2020/digital/news/buzzfeed-layoffs-twitter-am-to-dm-shutting-down-1234582127/